packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -397,15 +400,35 @@ export class ProjectActions {
     this.ctx.actions.electron.showBrowserWindow()
   }
 
+  get defaultE2EPath () {
+    const projectRoot = this.ctx.currentProject
+
+    assert(projectRoot, `Cannot create e2e directory without currentProject.`)
+
+    return path.join(projectRoot, 'cypress', 'e2e')
```

Comment:
would it be possible to pull this from packages/config so this default wouldn't need to be managed in two places?